packages/nextjs/src/config/wrappers/withSentryGetServerSideProps.ts

```diff
@@ -44,8 +44,9 @@ export function withSentryGetServerSideProps(
       if ('props' in serverSideProps) {
         const requestTransaction = getTransactionFromRequest(req);
         if (requestTransaction) {
+          const dynamicSamplingContext = requestTransaction.getDynamicSamplingContext();
           serverSideProps.props._sentryTraceData = requestTransaction.toTraceparent();
-          serverSideProps.props._sentryBaggage = serializeBaggage(requestTransaction.getBaggage());
+          serverSideProps.props._sentryBaggage = dynamicSamplingContextToSentryBaggageHeader(dynamicSamplingContext);
```

Comment:
nit: Would move down `const dynamicSamplingContext` to make it easier to read. Ditto to the rest of the nextjs wrappers.

```suggestion
          
          const dynamicSamplingContext = requestTransaction.getDynamicSamplingContext();
          serverSideProps.props._sentryBaggage = dynamicSamplingContextToSentryBaggageHeader(dynamicSamplingContext);
```

Reply:
5580981949f4cb0254879371fb728a5578842846 👍